### DISCUSSION TITLE
Plan for Macros?

### DISCUSSION THREAD

**Discussion by martinralfreindl** [Milad-Akarie/auto_route_library#2006] [Q&A]
Hey @Milad-Akarie, 

could you spare 2-3 sentences on your plans for macros? Seems like a super-exciting feature for auto_route, even though it's still in beta

Obviously no pressure if you're not planning to start on this anytime soon. Am just interested as a long-time enjoyer of auto_route :-)

**Comment by Milad-Akarie** [comment]
as soon as it becomes stable yes
